<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="routeLoaded">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="routeLoadedCancel_button">
       <property name="text">
        <string>Annuleer aanpassingen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <item>
        <widget class="QLabel" name="adresLabel">
         <property name="text">
          <string>Adres</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyLineEdit" name="lePostalAddress" native="true"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="zoomSpinBox">
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goButton">
         <property name="text">
          <string>Go</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QWebView" name="webView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="lwMarkers">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pbRemoveMarker">
       <property name="text">
        <string>Locatie verwijderen</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pbOptimizeRoute">
         <property name="text">
          <string>Route optimaliseren</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbShowRouteAsDefined">
         <property name="text">
          <string>Route tonen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRouteOmdraaien">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Route omdraaien</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbTransportationList">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Vervoerslijst aanmaken</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="totalWeightLabel">
         <property name="text">
          <string>Totaal gewicht:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="totalWeightEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalVolumeLabel">
         <property name="text">
          <string>Totaal volume:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="totalVolumeEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="totalBagsKurkLabel">
         <property name="text">
          <string>Lege zakken kurk:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="totalBagsKurkEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalBagsParafineLabel">
         <property name="text">
          <string>Lege zakken kaarsresten:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="totalBagsParafineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="showOphaalpunten">
       <property name="text">
        <string>Toon alle punten</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="totalKilometersLabel">
         <property name="text">
          <string>Totaal aantal kilometers:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="totalKilometersEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalTimeLabel">
         <property name="text">
          <string>Tijdsduur:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="totalTimeEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pbShowTotalDistanceAndTotalTime">
       <property name="text">
        <string>Bereken afstand en tijd</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>[[ For testing only ]]</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_latlng">
       <property name="text">
        <string>Zoek LAT en LNG</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QWidget</extends>
   <header>mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
